include/bluetooth/direction.h

```diff
@@ -26,6 +26,87 @@ struct bt_df_adv_cte_tx_param {
 	uint8_t  *ant_ids;
 };
 
+enum BT_DF_CTE_RX_TYPE {
+	BT_DF_CTE_RX_TYPE_AOA = BIT(0),
+	BT_DF_CTE_RX_TYPE_AOD_1US = BIT(1),
+	BT_DF_CTE_RX_TYPE_AOD_2US = BIT(2),
+};
+
+#define BT_DF_CTE_RX_TYPE_ALL                                                  \
+	(BT_DF_CTE_RX_TYPE_AOA | BT_DF_CTE_RX_TYPE_AOD_1US |                   \
+	 BT_DF_CTE_RX_TYPE_AOD_2US)
+
+/* @brief Constant Tone Extension parameters for connectionless
+ * reception.
+ *
+ * @note cte_type is a bit field that provides information about type of CTE
+ * an application expects:
+ * - BT_DF_CTE_RX_TYPE_AOA for Angle of Arrival
+ * - BT_DF_CTE_RX_TYPE_AOD_1US for Angle of Departure with 1us switching slots
+ * - BT_DF_CTE_RX_TYPE_AOD_2US for Angle of Departure with 2us switching slots
+ *
+ * In case BT_DF_CTE_RX_TYPE_AOA is not set, members: slot_durations,
+ * num_ant_ids and ant_ids are not required and their values will be not verified
+ * for correctness.
+ */
+struct bt_df_per_adv_sync_cte_rx_param {
+	/* Bitmap with allowed CTE types: AoD 1us, AoD 2us, AoA */
+	uint8_t cte_type;
+	/** Antenna switching slots, one of possible values: 1us, 2us. */
+	uint8_t slot_durations;
```

Comment:
How does the 1us and 2us here compare to the AoD 1us and AoD 2us?

Reply:
These values (1us and 2us) are related with AoA. In case `cte_type` has bit BT_DF_CTE_RX_TYPE_AOA unset. Then those values will be not verified in `bt_df_per_adv_sync_cte_rx_enable` call.

Reply:
Hmm, I'm just wondering if it's clear to the applicaiton what the possible values can actually be here. Since it's one of the two values, it doesn't make sense to expect values of the aforementioned enum. Is is such that 1 = 1us and 2 = 2us, or is it 0 = 1us, and 1 = 2us?

Reply:
I've created an enum with allowed values. I'll put a reference to the enum here.